TSPL.docc/ReferenceManual/Expressions.md

```diff
@@ -921,6 +923,101 @@ to make use of the implementation in their superclass.
 >
 > *superclass-initializer-expression* → **`super`** **`.`** **`init`**
 
+### Conditional Expression
+
+A *conditional expression* evaluates to one of several given values
+based on the value of a condition.
+It has one the following forms:
+
+```swift
+if <#condition 1#> {
+   <#expression used if condition 1 is true#>
+} else if <#condition 2#> {
+   <#expression used if condition 2 is true#>
+} else {
+   <#expression used if both conditions are false#>
+}
+
+switch <#expression#> {
+case <#pattern 1#>:
+    <#expression 1#>
+case <#pattern 2#> where <#condition#>:
+    <#expression 2#>
+default:
+    <#expression 3#>
+}
+```
+
+A conditional expression
+has the same behavior and syntax as an `if` statement or a `switch` statement,
+except for the differences listed below.
+
+A conditional expression appears only in the following contexts:
```

Comment:
```suggestion
except for the differences listed below.
A conditional expression appears only in the following contexts:
```

Reply:
I think this reads more clearly with the paragraph break — the "differences listed below" is all of the following paragraphs in this section, not just the unordered list that immediately follows.

Reply:
Readers (myself included) think that "below" is "right below", and not "throughout" a given section or series of examples. If that's the intent, please make that clearer.